app/components/article-comments.js

```diff
@@ -68,6 +68,14 @@ export default Component.extend(InViewportMixin, {
 
     this._super(...arguments);
 
+    const { urlThreadId } = this.articleComments.getUrlThreadParams();
+
+    if (this.isUcp && urlThreadId) {
+      this.articleComments.fetchCount(this.articleTitle, this.articleNamespace)
+      this.articleComments.load({ title: this.articleTitle, namespace: this.articleNamespace });
+      this.toggleProperty('isCollapsed');
```

Comment:
Are we sure we want to 'toggle' this instead of setting `true/false` directly?

Reply:
You're right, I will set this to `false`